### PR TITLE
update homepage description

### PR DESCRIPTION
**Pull request by cmaricle**

### CODE REVIEW

File: src/pages/index.jsx [hunk 1]
```diff
@@ -51,12 +56,13 @@ const MainPage = () => {
...
-							pupper waggy wags, shoob heckin good boys and girls borking doggo.
-							Maximum borkdrive very hand that feed shibe floofs shibe heckin
-							good boys and girls long doggo, I am bekom fat fat boi yapper
-							corgo. Doggorino long woofer the neighborhood pupper pupperino,
-							shoob. Very jealous pupper yapper clouds vvv, ur givin me a spook.
+							Just like any sound structure, our trackers are built with a solid
+							foundation. Our server runs in a AWS EC2 instance, using a
+							Postresql database with a Django backend. Our backend is the hub
+							for communication between our tracker and our frontend, providing
+							a RESTful API service. The frontend is written using in React and
```
Comment (Wyllan) [inline src/pages/index.jsx]: `The frontend is written using in React` remove either `using`, or `in`?

**Review by Wyllan — DISMISSED** [review]
Probably a good call lol. Approved in case you just want to push it, since text isn't breaking.

**Comment by bndnio** [comment]
probs good 🤷‍♀️

**Review by bndnio — DISMISSED** [review]

**Review by bndnio — APPROVED** [review]

**Comment by k-erby** [comment]
wow how dar u tak off abilitity 2 track huumans. RUDE >:( NOT APPROVE


But seriously, nice. I was going to bring up correcting it - glad you did! The entire app is looking dope, guys. :)

**Pull request merged**